bin/rose-check-software

```diff
@@ -220,7 +221,7 @@ def check(dependency, min_version=None, min_incompat_version=None, **kwargs):
     version_repr = dep_str(min_version, min_incompat_version)
     if version_repr:
         line += ' (%s)' % version_repr
-    line += ' ' + '.' * (TERM_WIDTH - len(line) - 25) + ' '
+    line += ' ' + '.' * (TERM_WIDTH - len(line) - 30) + ' '
```

Comment:
I don't remember changing this. Should I blame 2to3? Why would 2to3 do this?

Reply:
Good question! But no need to change if it works well for the rest of the app :+1:

Reply:
I honestly don't remember making this change, and I have no idea why I made it. Unless 2to3 did it for me (which seems unlikely). More than happy to revert if you think it important. Otherwise 'stet'.